build/Jenkinsfile

```diff
@@ -0,0 +1,138 @@
+#!/usr/bin/env groovy
```

Comment:
Just curious, is shebang needed? In the Jenkinsfile doc, shebang is not used.

https://jenkins.io/doc/book/pipeline/jenkinsfile/